tekton/debbuild/control/rules

```diff
@@ -14,8 +14,9 @@ BUILD_DATE := $(BUILD_DATE:+0000=Z)
 	dh $@
 
 override_dh_auto_build:
-	export XDG_CACHE_HOME=/tmp/cache ; \
-	test -e ./VERSION && VERSION=`cat VERSION` || VERSION=$(BUILD_DATE) ; \
+	@set -x ; export XDG_CACHE_HOME=/tmp/cache ; \
+	RELEASED_VERSION=`curl -s  https://api.github.com/repos/tektoncd/cli/releases/latest|python3 -c "import sys, json;x=json.load(sys.stdin);print(x['tag_name'])"|sed 's/^v//'` ; test -n "$$RELEASED_VERSION" || RELEASED_VERSION=$(BUILD_DATE) ; \
```

Comment:
This is scaring me a tiny bit. What if I rebuild the `deb` of an older version ? it will get the latest instead of the one I rebuild, isn't it ?

Reply:
There is no support for older version for releasing on ppt launchpad, it only supports latest and forward....

Reply:
and we already do that to get the latest tarball version, we could maybe support arbitrary version with some acrobatics shell scripts and argument passing but that makes much more complex to make.. why do y ou need to rebuild the deb of an older version?

Reply:
I don't know :) If a user want to build manually the `deb` from `tekton/debbuild` ?
It's a nit really, but :upside_down_face:

Reply:
If they want to build manually for a specific version they probably should just take the control directory and modify the `debian/rules` to their liking..